src/github.ts

```diff
@@ -584,7 +585,7 @@ export class GitHub {
         // it's easiest/safest to just pull this out by string search.
         const version = match[2];
         if (!version) continue;
-        if (prefix && match[1] !== prefix) {
+        if (prefix && match[1] !== packageBranch(prefix)) {
```

Comment:
could we drop the `@foo/` prefix before passing it into this method? I think the reason I didn't bump into this error is that I've been setting `packageName` when using the monorepo functionality.